test/unit/puppetlabs/services/jruby/jruby_puppet_service_test.clj

```diff
@@ -14,18 +13,24 @@
             [puppetlabs.trapperkeeper.testutils.logging :as logging]
             [puppetlabs.services.puppet-profiler.puppet-profiler-service :as profiler]
             [puppetlabs.services.jruby.jruby-puppet-core :as jruby-core]
```

Comment:
You didn't introduce this, but I notice we're requiring this ns twice as `jruby-puppet-core` and `jruby-core`